.changeset/silver-eels-rhyme.md

```diff
@@ -0,0 +1,6 @@
+---
+"@gql.tada/cli-utils": patch
+"gql.tada": patch
+---
+
+Fix vue losing out on the SFC plugin due to our usage of `getBasePlugins`
```

Comment:
suggesting to change this to make it obvious that `.vue` files were broken before

```suggestion
Fix Vue not transpiling to `.tsx` files properly due to missing SFC plugin
```